pkg/bff/api/v1/api.go

```diff
@@ -21,6 +21,7 @@ type BFFClient interface {
 	Login(context.Context) error
 
 	// Authenticated Endpoints
+	AddCollaborator(context.Context, *models.Collaborator) (*models.Collaborator, error)
```

Comment:
I'm not sure if this should be a separate request struct or not, since all that's currently required for collaborator creation is the email address.